CMakeLists.txt

```diff
@@ -73,6 +73,14 @@ set(CUSTOM_COMPILE_OPTIONS
     ""
     CACHE STRING "Allows adding custom C/C++ flags"
     )
+set(WUI
+    "YES"
+    CACHE BOOL "Enable Web User Interface"
+    )
+set(GUI
+    "YES"
+    CACHE BOOL "Enable GUI"
+    )
```

Comment:
This will break Private. Why not use PRINTERS_WITH_GUI list?

Reply:
my idea was that even MINI can be without GUI